packages/search/src/elasticsearch/dbhelper.ts

```diff
@@ -10,20 +10,19 @@
  */
 import { OPENCRVS_INDEX_NAME } from '@search/constants'
 import { ISearchResponse } from '@search/elasticsearch/client'
-import { ICompositionBody } from '@search/elasticsearch/utils'
+import { SearchDocument } from '@search/elasticsearch/utils'
 import { logger } from '@search/logger'
 import * as elasticsearch from '@elastic/elasticsearch'
 
 export const indexComposition = async (
   compositionIdentifier: string,
-  body: ICompositionBody,
+  body: SearchDocument,
   client: elasticsearch.Client
 ) => {
   let response: any
   try {
     response = await client.index({
       index: OPENCRVS_INDEX_NAME,
-      type: 'compositions',
```

Comment:
Future versions of ElasticSearch have deprecated this "type" thing. Cloning the index also caused the type not being cloned. I decided to remove it as we only save 1 type of data in search.